lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -130,7 +138,7 @@ class LowerLayersPass
     : public circt::firrtl::impl::LowerLayersBase<LowerLayersPass> {
   hw::OutputFileAttr getOutputFile(SymbolRefAttr layerName) {
     auto layer = dyn_cast<LayerOp>(
-        symbolTable->lookupSymbolIn(getOperation(), layerName));
+        SymbolTable::lookupSymbolIn(getOperation(), layerName));
```

Comment:
We ought to switch this pass over to use a SymbolTableCollection. In the meantime, I dropped the symbolTable because we weren't using it.